packages/core/build/build.tokens.ts

```diff
@@ -1,57 +1,155 @@
-import { baseTheme } from './tokens';
-import { Token, CdsTheme } from './token-utils';
+import * as ts from 'typescript';
 import * as fs from 'fs';
+import { baseTheme } from './tokens';
+import { Token } from './token-utils';
+
+const tokens: Token[] = Object.entries(flattenTokens(baseTheme)).map(token => token[1]);
+const experimental = `This token format is currently experimental and may change in the future`;
+
+fs.mkdirSync('../dist/core/styles/', { recursive: true });
+fs.mkdirSync('../dist/core/tokens/', { recursive: true });
+fs.mkdirSync('../src/styles/tokens/generated', { recursive: true });
+
+// Public API Tokens
+buildCSSTokens('../dist/core/styles/module.tokens.css');
+buildJSONTokens('../dist/core/tokens/tokens.json');
+buildJSTokens('../dist/core/tokens/tokens.ts');
+buildAndroidXMLTokens('../dist/core/tokens/tokens.android.xml');
+buildIOSSwiftTokens('../dist/core/tokens/tokens.ios.swift');
+buildSassTokens('../dist/core/tokens/tokens.scss');
+
+// Internal API Tokens for custom elements with fallback values
+buildInternalSassTokens('../src/styles/tokens/generated/_index.scss');
```

Comment:
The internal sass tokens provide the vars with a inlined fallback in case the global tokens fail. 
`var(--token-color, #fff)`